Index: dwm_patches/dwm/config.def.h
===================================================================
--- dwm_patches.orig/dwm/config.def.h
+++ dwm_patches/dwm/config.def.h
@@ -51,6 +51,7 @@ static const Layout layouts[] = {
 #define XF86XK_AudioRaiseVolume     0x1008ff13
 #define XF86XK_AudioLowerVolume     0x1008ff11
 #define XF86XK_AudioMicMute         0x1008FFB2
+#define XF86XK_PowerOff             0x1008ff2a
 #define MODKEY Mod4Mask
 #define TAGKEYS(KEY,TAG) \
 	{ MODKEY,                       KEY,      view,           {.ui = 1 << TAG} }, \
@@ -85,7 +86,7 @@ static Key keys[] = {
 	{ MODKEY,                       XK_l,      setmfact,       {.f = +0.05} },
 	{ MODKEY|ShiftMask,             XK_space,  zoom,           {0} },
 	{ MODKEY,                       XK_Tab,    view,           {0} },
-	{ MODKEY|ShiftMask,             XK_c,      killclient,     {0} },
+	{ MODKEY,                       XK_q,      killclient,     {0} },
 	{ MODKEY,                       XK_t,      setlayout,      {.v = &layouts[0]} }, // tile
 	{ MODKEY,                       XK_m,      setlayout,      {.v = &layouts[1]} }, // monocle
 	{ MODKEY,                       XK_u,      setlayout,      {.v = &layouts[2]} }, // centeredmaster
@@ -116,7 +117,8 @@ static Key keys[] = {
 	TAGKEYS(                        XK_7,                      6)
 	TAGKEYS(                        XK_8,                      7)
 	TAGKEYS(                        XK_9,                      8)
-	{ MODKEY|ShiftMask,             XK_q,      quit,           {0} },
+	{ MODKEY|ShiftMask,             XK_q,      spawn,          SHCMD("sysact") },
+	{ MODKEY,               XK_BackSpace,      spawn,          SHCMD("sysact") },
 	{ MODKEY|ShiftMask,             XK_F1,     tagall,         {.v = "F1" } },
 	{ MODKEY|ShiftMask,             XK_F2,     tagall,         {.v = "F2" } },
 	{ MODKEY|ShiftMask,             XK_F3,     tagall,         {.v = "F3" } },
@@ -132,6 +134,7 @@ static Key keys[] = {
 	{ 0, XF86XK_AudioRaiseVolume,  spawn, SHCMD("amixer -q sset Master 5%+") },
 	{ 0, XF86XK_AudioLowerVolume,  spawn, SHCMD("amixer -q sset Master 5%-") },
 	{ 0, XF86XK_AudioMicMute,      spawn, SHCMD("amixer -q sset Capture toggle") },
+	{ 0, XF86XK_PowerOff,          spawn, SHCMD("sysact") },
 };
 
 /* button definitions */
